__tests__/app.test.js

```diff
@@ -212,10 +213,147 @@ describe('PATCH requests testing', () => {
+                .post('/api/articles/666/comments')
+                .send(requestConntentToPost)
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+        })
+
+        test('Error 400 when article_id is not valid ', () => {
+            return request(app)
+                .post('/api/articles/not-valid-id/comments')
+                .send(requestConntentToPost)
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+
+        });
+
+        test('Error 400 when passed body is empty ', () => {
+            return request(app).
+                post('/api/articles/3/comments')
+                .send({})
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+
+
+        })
+
+        test('Error 400 when passed body has only one property ', () => {
+            return request(app)
+                .post('/api/articles/3/comments')
+                .send(
+                    {
+                        body: 'Just looking'
+                    })
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+
+        })
+
+        test('Error 400 when passed body has more then two properties ', () => {
+            return request(app)
+                .post('/api/articles/3/comments')
+                .send(
+                    {
+                        username: 'lurker',
+                        body: 'Just looking',
+                        num: 11
+                    }
+                )
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+
+        })
+
+        test('Error 400 when username value is not a string ', () => {
+            return request(app)
+                .post('/api/articles/3/comments')
+                .send(
+                    {
+                        username: 33,
+                        body: 'Just looking',
+                    }
+                )
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+
+
+        })
+
+        test('Error 400 when passed body value is not a string ', () => {
+            return request(app)
+                .post('/api/articles/3/comments')
+                .send(
+                    {
+                        username: 33,
+                        body: 'Just looking',
+                    }
+                )
+                .expect(400)
+                .then(({ body: { msg } }) => {
+                    expect(msg).toBe("Bad request")
+                })
+        })
+    });
```

Comment:
This is a duplicated test: It should be the body property which is not a string

Also we could do with test for if the keys are wrong eg {uzzername: 'kev'} or {bozy: 'no comment'}